test/factoids_test.coffee

```diff
@@ -144,12 +145,24 @@ describe 'factoids', ->
 
       adapter.receive(new TextMessage user, 'hubot: remember foo')
 
-  it 'responds to factoids', (done) ->
+  describe 'list all factoids', ->
+    beforeEach ->
+      robot.brain.data.factoids.foo = value: 'bar', forgotten: true
+      robot.brain.data.factoids.bas = value: 'baz', forgotten: false
+
+    it 'responds to list all factoids', (done) ->
+      adapter.on 'reply', (envelope, strings) ->
+        expect(strings[0]).to.match /All factoids: \n!bas: baz\n/
+        done()
+
+      adapter.receive(new TextMessage user, 'hubot: list all factoids')
+
+  it 'responds to list all factoids', (done) ->
```

Comment:
That's fine, if you want to add it, but it looks like you removed the test for the URL?